scripts/build/rollup.config.js

```diff
@@ -54,6 +55,39 @@ export const umd = {
   ],
 };
 
+const updateLibPrefix = (isUndo) => {
+  const path = 'src/lib-prefix.ts'
+  if (!fs.existsSync(path)) {
+    // Supported in rollup 4, we're currently rollup 2
+    // this.error(`File not found: ${path}`);
+    return;
+  }
+
+  let content = fs.readFileSync(path, 'utf-8');
+  let updatedContent;
+  if (isUndo) {
+    updatedContent = content.replace(/amplitude-ts-sdk-script/g, 'amplitude-ts');
+  } else {
+    updatedContent = content.replace(/amplitude-ts/g, 'amplitude-ts-sdk-script');
+  }
+
+  fs.writeFileSync(path, updatedContent, 'utf-8');
+  // this.info(`File updated: ${path}`);
```

Comment:
Ditto

Reply:
the same as above https://github.com/amplitude/Amplitude-TypeScript/pull/939#discussion_r1896979850

Reply:
Nit: Should we also add the comment `// Supported in rollup 4, we're currently rollup 2` here for clarity